app/helpers/application_helper/button/basic.rb

```diff
@@ -42,9 +42,15 @@ def localized(key, value = nil)
   #   self[:prompt]  -- the user has to confirm the action
   #   self[:hidden]  -- the button is not displayed in the toolbar
   #   self[:text]    -- text for the button
+  #   self[:onwhen]  -- enable button if items are selected
   def calculate_properties
-    self[:enabled] = !disabled? if self[:enabled].nil?
-    self[:title] = @error_message if @error_message.present?
+    enabled = !disabled?
+    self[:enabled] = enabled if self[:enabled].nil?
+
+    unless self[:enabled] || enabled
```

Comment:
| :enable | disabled? | :enabled | :title | :onwhen |  
|-----------|--------------|------------|-------|-------------|
| true | true/false | unchanged | unchenged | unchanged |
| false | true | false | error | nil |
| false | false | false | unchenged | nil |
| nil | true | false | error | nil |
| nil | false | true | unchenged | unchanged |

@romanblanco 
we change `:enable` only if it's nil
but we change :title and :onwhen on nil _or_ false

but yes it's not nice, I can't change is to `.nil?` but can we do something else to make it better ?
my main problem is the :enable == false is not really false it's false unless :onwhen is true :-(